src/react/index.ts

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import {provideReactWrapper} from '@microsoft/fast-react-wrapper';
+import {provideVSCodeDesignSystem} from '../vscode-design-system';
+
+import {
+	vsCodeBadge,
+	vsCodeButton,
+	vsCodeCheckbox,
+	vsCodeDataGrid,
+	vsCodeDataGridCell,
+	vsCodeDataGridRow,
+	vsCodeDivider,
+	vsCodeDropdown,
+	vsCodeLink,
+	vsCodeOption,
+	vsCodePanels,
+	vsCodePanelTab,
+	vsCodePanelView,
+	vsCodeProgressRing,
+	vsCodeRadio,
+	vsCodeRadioGroup,
+	vsCodeTag,
+	vsCodeTextArea,
+	vsCodeTextField,
+} from '..';
+
+const {wrap} = provideReactWrapper(React, provideVSCodeDesignSystem());
+
+export const VSCodeBadge = wrap(vsCodeBadge());
+export const VSCodeButton = wrap(vsCodeButton());
+export const VSCodeCheckbox = wrap(vsCodeCheckbox());
+export const VSCodeDivider = wrap(vsCodeDivider());
+export const VSCodeLink = wrap(vsCodeLink());
+export const VSCodeProgressRing = wrap(vsCodeProgressRing());
+export const VSCodeTag = wrap(vsCodeTag());
+export const VSCodeTextArea = wrap(vsCodeTextArea());
+export const VSCodeTextField = wrap(vsCodeTextField());
+export const VSCodeOption = wrap(vsCodeOption());
+export const VSCodeDropdown = wrap(vsCodeDropdown());
+export const VSCodePanels = wrap(vsCodePanels());
+export const VSCodePanelTab = wrap(vsCodePanelTab());
+export const VSCodePanelView = wrap(vsCodePanelView());
+export const VSCodeRadio = wrap(vsCodeRadio());
+export const VSCodeRadioGroup = wrap(vsCodeRadioGroup());
+export const VSCodeDataGrid = wrap(vsCodeDataGrid());
+export const VSCodeDataGridCell = wrap(vsCodeDataGridCell());
+export const VSCodeDataGridRow = wrap(vsCodeDataGridRow());
```

Comment:
I also just remembered that we still need to configure component events for most of these. Have you done any work on that yet?

https://www.fast.design/docs/integrations/react/#configuring-custom-events

Reply:
I have not. I assumed we were still blocked on this, considering native DOM events don't even seem to be working. For example 👇 

```ts
<VSCodeCheckbox
  checked // TypeError, also doesn't affect the state of the checkbox 
  onChange={(e) => {
    console.log(e); // never fires
  }}
>
  Subscribe
</VSCodeCheckbox>
```

Reply:
Yep sounds great, we can hold off on this until a resolution to [this issue](https://github.com/microsoft/fast/issues/5387) comes down the line.